src/main/java/me/earth/earthhack/impl/core/mixins/entity/living/MixinEntityLivingBase.java

```diff
@@ -429,6 +439,20 @@ public void swingArmHook(EnumHand hand, CallbackInfo ci)
         }
     }
 
+    @Inject(method = "getArmSwingAnimationEnd", at = @At("HEAD"), cancellable = true)
+    public void getArmSwingAnimationEnd(CallbackInfoReturnable<Integer> cir) {
```

Comment:
```suggestion
    public void getArmSwingAnimationEndHook(CallbackInfoReturnable<Integer> cir) {
```
This is a mixin method, it does not really matter what its named

Reply:
No I meant for getArmSwingAnimationEnd mapping, it conflicts.
Here:
Mapping conflict for @Invoker target getArmSwingAnimationEnd: func_82166_i()I for target net.minecraft.entity.EntityLivingBase conflicts with existing mapping Lnet/minecraft/entity/EntityLivingBase;func_82166_i()I